### ISSUE TITLE
Parse Geometry

### ISSUE THREAD

**Issue by jeromerolland** [jmirtsch/GeometryGymIFC#13]
Hi,
do you have an example for parsing geometry in ifc file ?

best regards,
Jérôme

**Comment by jeromerolland** [comment]
Hi Jon,

Thank you very much for your code sample.

Best regards,

Jérôme

**Comment by CJAndrew84** [comment]
Jon

Has this code example changed? I'm trying to use it at the moment and it's not liking "Bounds[0].Bound"

Chris

**Comment by jmirtsch** [comment]
Yes, I've been improving the plugin to use a "SET" (unordered collection) rather than a list when the IFC specification nominates it.  You can use linq using "Bounds.First().Bound" instead of a zero index.  Sorry for the breaking change, but it does improve the toolkit in other aspects.
